docs/sources/developers/plugins/metadata.md

```diff
@@ -21,22 +21,28 @@ The plugin.json file is required for all plugins. When Grafana starts, it scans
+| `$schema`       | string                  | No       | Schema definition for the plugin.json file                                                                                                                                                                                                                                                                                                                                                              |
 | `alerting`      | boolean                 | No       | For data source plugins. If the plugin supports alerting.                                                                                                                                                                                                                                                                                                                                               |
 | `annotations`   | boolean                 | No       | For data source plugins. If the plugin supports annotation queries.                                                                                                                                                                                                                                                                                                                                     |
 | `backend`       | boolean                 | No       | If the plugin has a backend component.                                                                                                                                                                                                                                                                                                                                                                  |
-| `category`      | string                  | No       | Plugin category used on the Add data source page. Possible values are: `tsdb`, `logging`, `cloud`, `tracing`, and `sql`.                                                                                                                                                                                                                                                                                |
+| `baseUrl`       | string                  | No       |                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
i don't think this should be set explicitly (but not totally sure) i think it is based on where the plugin is running:  ie the appUrl etc